src/watcher/INotifyWatcher.zig

```diff
@@ -94,9 +94,7 @@ pub fn init(this: *INotifyWatcher, _: []const u8) !void {
     bun.assert(!this.loaded);
     this.loaded = true;
 
-    if (bun.getenvZ("BUN_INOTIFY_COALESCE_INTERVAL")) |env| {
-        this.coalesce_interval = std.fmt.parseInt(isize, env, 10) catch 100_000;
-    }
+    this.coalesce_interval = @intCast(bun.EnvVar.bun_inotify_coalesce_interval.get());
```

Comment:
Previously, `BUN_INOTIFY_COALESCE_INTERVAL=9223372036854775808` would cause `this.coalesce_interval` to be 100,000. Now it invokes illegal behavior. We should probably not add new illegal behavior.